tests/newfeatures/drush.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Running Drush Commands on a Drupal Site
+  In order to interact with Drupal without configuring Pantheon site aliases
+  As a Terminus user
+  I want the ability to run arbitrary drush commands in terminus
+
+  Background: I am authenticated and have a site named [[test_site_name]]
+    Given I am authenticated
+    And a site named: [[test_site_name]]
+
+  Scenario: Running a simple drush command
```

Comment:
The feature test is still isn't being run. It needs to be moved into the `tests/active_features` directory. I can't get this command to work for myself.